diploma-project/pages/index.jsx

```diff
@@ -2,9 +2,13 @@ import React from "react";
 import Head from 'next/head';
 import Navbar from "../components/Navbar";
 import Footer from "../components/Footer";
-import Carousel from "../components/Carousel";
-import ProductCard from "../components/ProductCard";
-import AvatarIcon from "../components/AvatarIcon";
+import ItemsScroll from "../components/ItemsScroll";
```

Comment:
You can group these imports. Create an index.js file in components. It should look something like this:

```
export { default as Navbar } from './Navbar'
export { default as Footer } from './Footer'
export { default as ItemsScroll } from './ItemsScroll'
```


Then you can import everything you need in one line here
`import { Navbar, Footer, ItemsScroll } from './components'`